src/test/java/org/apache/sling/distribution/agent/impl/PrivilegeDistributionRequestAuthorizationStrategyTest.java

```diff
@@ -36,19 +36,21 @@
  */
 public class PrivilegeDistributionRequestAuthorizationStrategyTest {
 
+    String jcrPrivilege = "foo";
+    String[] additionalJcrPrivilegesForAdd;
```

Comment:
consider removing fields `jcrPrivilege`, `additionalJcrPrivilegesForAdd` and `additionalJcrPrivilegesForDelete`. Restore earlier, method-local `jcrPrivilege` and pass `null` directly while intiallizing `PrivilegeDistributionRequestAuthorizationStrategy` in the existing tests asserting existing behaviour (of adding default JCR_READ/JCR_REMOVE_NODE privileges as applicable)